### PR TITLE
Django 32

### PR DESCRIPTION
**Pull request by ccurvey**
second attempt at django-32 PR

### CODE REVIEW

File: test_project/settings.py [hunk 1]
```diff
@@ -79,7 +79,9 @@
     'django.contrib.sites',
     'django.contrib.admin',
     'django.contrib.messages',
-    
+
     'jsonate',
     'test_app',
 )
+
+DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
```
Comment (igorkramaric) [inline test_project/settings.py:87]: Ah, this one should actually be `DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'` as per MP's [#2736](https://github.com/mediapredict/mediapredict/issues/2736#issue-981507834).
This way the only migration we'll do will be the one regarding BooleanField

File: test_project/test_app/models.py [hunk 2]
```diff
@@ -10,21 +10,21 @@ class MyModel(models.Model):
...
-    
+
     file_field = models.FileField(upload_to="files")
     image_field = models.ImageField(upload_to="images")
-    
+
     boolean_field = models.BooleanField(default=True)
-    null_field = models.NullBooleanField(default=None)
-    
+    null_field = models.BooleanField(default=None, null=True)
```
Comment (igorkramaric) [inline test_project/test_app/models.py:26]: nice catch

**Review by igorkramaric — CHANGES_REQUESTED** [review]
This is good to go, except for one comment.

File: test_project/settings.py (hunk 1 above)
Comment (ccurvey) [reply test_project/settings.py]: ohhhh...I thought Django was smart enough to only apply that value to new tables.  I'm thinking of some other projects where that migration would have been a disaster.

**Review by igorkramaric — APPROVED** [review]

**Pull request merged**